package.json

```diff
@@ -37,20 +37,20 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "autoprefixer": "^9.7.4",
+    "css-mqpacker": "^7.0.0",
+    "cssnano": "^4.1.11",
```

Comment:
`npm install` automatically alphabetizes dependencies in `package.json`. I'm not certain why this hadn't occurred previously, but we'll take care of it in this commit.